src/main.js

```diff
@@ -100,8 +100,18 @@ function init(i18next, options={}){
         }
     }
 
-    function handle(selector, opts){
-        var elems = options.document.querySelectorAll(selector);
+    function handle(selectorOrElements, opts){
+        var elems;
+        if(typeof selectorOrElements === 'string') {
+            // Assume query selector
+            elems = options.document.querySelectorAll(selectorOrElements);
+        } else if(Array.isArray(selectorOrElements)) {
```

Comment:
Maybe this should be extended to support all kinds of iterable objects like `NodeList` etc.